unit_tests/kubernetes_installer_test.py

```diff
@@ -1,16 +1,105 @@
+from mock import patch
+from path import path
```

Comment:
Great find @chuckbutler .  Can this path mangling be done with the lower case path?  My understanding was the Path() object was needed to add the hooks directory to the python path.  

If there is a different way that I am not aware of please let me know.